modules/docs/arrow-docs/docs/_data/menu.yml

```diff
@@ -195,6 +195,9 @@ options:
       - title: Bimonad
         url: /docs/typeclasses/bimonad/
 
+      - title: Bifunctor
+        url: /docs/typeclasses/bifunctor
+
```

Comment:
Can you please add a short description and a link in the typeclasses/intro doc?

Reply:
Sure 😄